README.md

```diff
@@ -0,0 +1,210 @@
+# 📙Online Book Store
+Welcome to the Online Book Store project! This application is designed to provide a seamless experience 
+for users to browse, purchase, and manage books, while also offering administrative functionalities for managers. 
+This project was developed using Spring Boot and includes various functionalities for both shoppers and managers.
+
+## 📅 Table of Contents
+
+- [Project Description](#project-description)
+- [Technologies Used](#technologies-used)
+- [Features](#features)
+  - [User](#user)
+  - [Admin](#admin)
+- [Setup Instructions](#setup-instructions)
+- [Postman Collection](#postman-collection)
+- [Functionality](#functionality)
+- [Author](#author)
+
+## ⭐ Project Description
+
+The Online Book Store is a web application that allows users to browse and purchase books online. 
+Users can search for books, view book details, add books to their shopping cart, and complete purchases. 
+Admin users can manage the inventory, organize bookshelf sections, and update order statuses. 
+This project aims to simulate a real-world online book store and demonstrate the capabilities of a Spring Boot application.
+
+## ⚙️ Technologies Used
+
+- **Spring Boot**: Core framework for the application
+- **Spring Security**: For authentication and authorization
+- **Spring Data JPA**: For database interactions
+- **Swagger**: API documentation
+- **Lombok**: Reducing boilerplate code
+- **Maven**: Dependency management
+- **Checkstyle**: Code style enforcement
+- **Liquibase**: Database management
+- **Mockito**: Mocking framework for tests
+- **MySQL**: Database management system
+- **JUnit**: Testing framework
+- **Docker**: Deployment and scalability
+- **Testcontainers**: Integration testing with Docker containers
+- **CI/CD**: Continuous Integration setup with GitHub Actions
+
+## ⛓ Features
+
+### 🖥️ User
+
+- **Join and Sign In**: Users can register and log in to the application.
+- **Browse Books**: View all books, search by name, and view details of a specific book.
+- **Shopping Cart**: Add books to the cart, view the cart, and remove books from the cart.
+- **Purchase Books**: Complete the purchase of books in the cart.
+- **Order History**: View past orders and their details.
+
+### 🤖 Admin
+
+- **Manage Books**: Add, update, and remove books from the store.
+- **Organize Sections**: Create, update, and delete bookshelf sections.
+- **Manage Orders**: View and update the status of orders.
+
+## Setup Instructions
+### Installation without Docker
```

Comment:
Please add preconditions section to give some information about what should be installed and which versions of dependencies should be